eyeglass-exports.js

```diff
@@ -0,0 +1,8 @@
+const path = require('path');
```

Comment:
Just realised something: The `srcSassDir` returned by `index.js` is already an absolute path. Using `path` to join it with `__dirname` is therefore redundant here and can be removed. Sorry I didn't spot this earlier!

Reply:
👌